leo/commands/new.rs

```diff
@@ -14,81 +14,59 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    cli::*,
-    cli_types::*,
-    errors::{CLIError, NewError},
-};
-use leo_package::LeoPackage;
+use crate::{commands::Command, context::Context};
+use anyhow::{anyhow, Result};
 
-use clap::ArgMatches;
+use leo_package::LeoPackage;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates